indico/modules/rb/client/js/common/timeline/TimelineItem.jsx

```diff
@@ -138,8 +138,16 @@ class TimelineItem extends React.Component {
     return (segStartMins / ((endHour - startHour) * 60)) * 100;
   };
 
-  renderMessagePopup = (message, segmentStartDt, segmentEndDt) => {
+  renderMessagePopup = (message, segmentStartDt, segmentEndDt, reservation) => {
     const {dayBased} = this.props;
+    const mapRecurrenceTypeToInfo = type =>
+      type.toLowerCase() === 'day'
+        ? Translate.string('daily')
+        : type.toLowerCase() === 'week'
+        ? Translate.string('weekly')
+        : type.toLowerCase() === 'month'
+        ? Translate.string('monthly')
+        : '';
```

Comment:
A simple dict would be much easier here than a function ;) But in any case, as @tomasr8 mentioned it would be better to have 3 separate i18n strings so they can be properly translated.

Reply:
I think you also need to take the repeatInterval into account - right now the tooltip wouldn't be correct for something that repeats e.g. every 2 weeks